### PR TITLE
Add execution model endpoint

### PR DESCRIPTION
**Pull request by r00ta**

### CODE REVIEW

File: end2endPoC/kogito-consumer/app/src/main/java/com/redhat/developer/execution/responses/ExecutionModelResponse.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+    public ExecutionModelResponse(){}
+
+    public static ExecutionModelResponse fromDmnModel(DmnModel model){
+        ExecutionModelResponse response = new ExecutionModelResponse();
+        response.deploymentDate = null;
+        response.modelId = model.modelId;
+        response.name = model.name;
+        response.namespace = model.nameSpace;
+        response.type = "http://www.omg.org/spec/DMN/20151101/dmn.xsd";
```
Comment (r00ta) [inline end2endPoC/kogito-consumer/app/src/main/java/com/redhat/developer/execution/responses/ExecutionModelResponse.java:40]: hardcoded for now

File: end2endPoC/kogito-consumer/app/src/main/java/com/redhat/developer/execution/responses/ExecutionModelResponse.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        response.model = model.model;
+        response.serviceIdentifier = new ServiceIdentifier();
+        return response;
```
Comment (r00ta) [inline end2endPoC/kogito-consumer/app/src/main/java/com/redhat/developer/execution/responses/ExecutionModelResponse.java:43]: Some properties are hardcoded atm

File: end2endPoC/kogito-consumer/app/src/main/java/com/redhat/developer/execution/api/DecisionsApi.java [hunk 3]
```diff
@@ -209,6 +212,28 @@ public Response getExecutionGraphOutcomes(
...
+            @APIResponse(description = "Gets the model that evaluated the execution.", responseCode = "200", content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(type = SchemaType.OBJECT, implementation = ExecutionModelResponse.class))),
+            @APIResponse(description = "Bad Request", responseCode = "400", content = @Content(mediaType = MediaType.TEXT_PLAIN))
+    }
+    )
+    @Operation(summary = "Gets the model that evaluated the execution.", description = "Gets the model that evaluated the execution.")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getDecisionModel(
+            @Parameter(
+                    name = "key",
+                    description = "ID of the decision that needs to be fetched",
```
Comment (manstis) [inline end2endPoC/kogito-consumer/app/src/main/java/com/redhat/developer/execution/api/DecisionsApi.java:227]: "ID of the execution....."?

Comment (r00ta) [reply end2endPoC/kogito-consumer/app/src/main/java/com/redhat/developer/execution/api/DecisionsApi.java]: Fixed :)

File: end2endPoC/kogito-consumer/app/src/main/java/com/redhat/developer/execution/responses/ExecutionModelResponse.java [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    public ExecutionModelResponse(){}
+
+    public static ExecutionModelResponse fromDmnModel(DmnModel model){
+        ExecutionModelResponse response = new ExecutionModelResponse();
+        response.deploymentDate = null;
+        response.modelId = model.modelId;
+        response.name = model.name;
+        response.namespace = model.nameSpace;
+        response.type = "http://www.omg.org/spec/DMN/20151101/dmn.xsd";
+        response.model = model.model;
```
Comment (manstis) [inline end2endPoC/kogito-consumer/app/src/main/java/com/redhat/developer/execution/responses/ExecutionModelResponse.java:41]: Yay, once https://github.com/kiegroup/kogito-tooling/pull/142 is merged and I make the subsequent changes in trusty I can use this to populate the viewer :-)

**Pull request merged**